<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>427</width>
    <height>430</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="lb_sendingData">
      <property name="text">
       <string>Введите данные для отправки</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="te_getText"/>
    </item>
    <item>
     <widget class="QLabel" name="lb_receiveData">
      <property name="text">
       <string>Принятые данные</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="te_result">
      <property name="inputMethodHints">
       <set>Qt::ImhMultiLine</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pb_clearResult">
      <property name="text">
       <string>Очистить все окна </string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pb_sendDataG">
      <property name="text">
       <string>Отправить датаграмму</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pb_stop">
      <property name="text">
       <string>Остановить передачу</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pb_start">
      <property name="text">
       <string>Начать передачу</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
